tasks/upload-assets-to-s3.js

```diff
@@ -30,7 +30,7 @@ function getFileEncoding (filename) {
 }
 
 async function uploadFile (file, opts, s3) {
-	const basename = path.basename(file);
+	const basename = file.replace(/^.+?[/]/, ''); // remove first directory only
```

Comment:
Might be cleaner and clearer? Regex always takes a bit of mental picking apart.
```suggestion
	const basename = file..split('/').splice(1).join('/'); // remove first directory only
```